src/app/econ/supply-demand/lesson/SupplyDemandLesson.tsx

```diff
@@ -0,0 +1,807 @@
+              </Collapsible>
+            </li>
+          </ul>
+        </div>
+      </ColorBox>
+      <ColorBox color="purple">
+        <p>
+          <Emphasize>
+            What is the difference between demand and quantity demanded?
+          </Emphasize>
+        </p>
+        <p>
+          Demand is the entire relationship between prices and how much people
+          want at each price. Quantity demanded is how much people want at one
+          particular price.
+          <br />
+          <br />
+          Therefore, the change in demand shifts the{" "}
+          <Emphasize>ENTIRE</Emphasize> curve, while the change in quantity
+          demanded changes <Emphasize>ALONG</Emphasize> the curve.
+        </p>
+      </ColorBox>
+
+      <p>So let's try this out in the interactive tool below!</p>
+      <br />
+
+      <iframe
+        title="supply and demand interactive tool"
+        src="https://dev.pickcode.io/lesson/supply-and-demand-lesson-cmce1y3ab02zmk3y1a39dulwh-2025-07-09-01-22-14"
+        width="100%"
+        height="650px"
+      ></iframe>
+      <ColorBox color="purple">
+        As you can see, changes in price{" "}
+        <Emphasize>moves along the curve</Emphasize>
+      </ColorBox>
+      <ColorBox color="blue">
+        <p>
+          <Emphasize>👉 Check in: </Emphasize>
+        </p>
+
+        <QuizQuestion
+          question="As price falls, what happens to the quantity demanded?"
+          choices={[
+            {
+              text: "Quantity demanded stays the same",
+              isCorrect: false,
+              explanation:
+                "Oops! Try running the tool again! What do you notice?",
+            },
+            {
+              text: "Quantity demanded goes down",
+              isCorrect: false,
+              explanation:
+                "Oops! Try running the tool again! What do you notice?",
+            },
+            {
+              text: "Quantity demanded goes up",
+              isCorrect: true,
+              explanation: "Correct!",
+            },
+            {
+              text: "Quantity demanded goes up and down",
+              isCorrect: false,
+              explanation:
+                "Oops! Try running the tool again! What do you notice?",
+            },
+          ]}
+        ></QuizQuestion>
+      </ColorBox>
+      <ColorBox color="purple">
+        <p>
+          Make sure you write down your code for the for loop that goes along
+          the supply curve!
+        </p>
+      </ColorBox>
+    </Block>,
+
+    <Block color="blue" title="Demand">
+      <ColorBox color="yellow">
+        <ul>
+          <li>
+            <Emphasize>Demand Shifters</Emphasize>
+          </li>
+          <li>
+            <Collapsible
+              ExampleContent={
+                <>
+                  <Emphasize>
+                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
```

Comment:
I see that you have to put a lot of space characters here to tab content...
Instead, why not change the styling in the Collapsible component to just have padding-left. You shouldn't have to add any leading spaces.

Reply:
This will also fix the issue that multi-line text doesn't adhere to the spacing you set up manually

<img width="774" height="133" alt="Screenshot 2025-07-16 at 4 45 03 PM" src="https://github.com/user-attachments/assets/7c078374-2825-43cc-974a-bc0fb5548221" />